tests/test_expectation_maximization.py

```diff
@@ -158,10 +158,13 @@ def test_compute_fsc(test_ir, n_pix):
 
 def test_expand_1d_to_3d(test_ir, n_pix):
```

Comment:
Yeah I'm not sure what's going on with the tests in this branch - the branch contains the test files from PR #21 (this is a branch off of itr_ref_docstrings), but the tests aren't co-operating.

Reply:
Update to this - since this is being pushed to itr_ref_docstrings which doesn't appear in the "pull_request" sections of test.yml and lint.yml, the actions don't run. This will not be an issue when merging this branch into master.

Reply:
This is a reason to have a dev branch. 
* Branch dev off of itr_ref_docstrings. 
* PR itr_ref_1d_to_3d into dev
* tests run since dev has tests

Reply:
Ah I see, good argument thank you!